CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 Changelog
 =========
 
+2023-09-25 v2.7:
+
+* add: compatibility with PostgreSQL 15 and 16, Jehan-Guillaume de Rorthais, Thomas Reiss
+* change: `stat_snapshot_age` is compatible from PostgreSQL v9.5 to v14, Benoit Lobréau
+* change: simplify session accounting in `backends_status`, Thomas Reiss
+* fix: in `pg_dump_backup`, error explicitly if given `--path` is a directory, Christophe Courtois
```

Comment:
Not really a fix, it was working as intented, that's just user-friendliness

```suggestion
* in `pg_dump_backup`, error explicitly if `--path` is a directory, Christophe Courtois
```

Reply:
Well, we need a label, either `new`, `change` or `fix`, or manque your own :)

I decided to use `fix` as the error message was (probably) wrong before...